contents/preface.md

```diff
@@ -78,3 +78,5 @@ Many people have contributed directly and indirectly to this book.
 Jose Storopoli would like to thank his colleagues, specially [Fernando Serra](https://orcid.org/0000-0002-8178-7313), [Wonder Alexandre Luz Alves](https://orcid.org/0000-0003-0430-950X) and [André Librantz](https://orcid.org/0000-0001-8599-9009), for their encouragement and support.
 
 Rik Huijzer would like to thank his PhD supervisors, [Peter de Jonge](https://www.rug.nl/staff/peter.de.jonge/), [Ruud den Hartigh](https://www.rug.nl/staff/j.r.den.hartigh/) and [Frank Blaauw](https://frankblaauw.nl/), for their support.
+
+Lazaro Alonso would like to thank his wife and daughters for their encouragement to get involved in this project. 
```

Comment:
Super nice to read :smile: I hope we can make something which they will like in the end